pages/common/git-credential.md

```diff
@@ -0,0 +1,16 @@
+# git credential
+
+> Retrieve and store user credentials.
+> More information: <https://git-scm.com/docs/git-credential>.
+
+- Display credential information, retrieving the username/password from config files:
+
+`echo {{"url=http://example.com"}} | git credential fill`
+
+- Send credential information to configured credential helpers to store for later use:
+
+`echo {{"url=http://example.com"}} | git credential approve`
+
+- Send credential information to configured credential helpers to erase:
```

Comment:
Perhaps this will read better?
```suggestion
- Erase the specified credential information from all the configured credential helpers:
```

...or would it change the meaning too much?

Reply:
Not at all.